documentation/md/style.md

```diff
@@ -619,6 +619,7 @@ Border:
  * **`text-border-width`** : The width of the border around the label.
  * **`text-border-style`** : The style of the border around the label; may be `solid`, `dotted`, `dashed`, or `double`.
  * **`text-border-color`** : The colour of the border around the label.
+ * **`text-border-radius`**: The radius of the border around the label. This will override the `text-background-shape` property.
```

Comment:
It would be best to restrict the application of `text-border-radius` to `text-background-shape: round-rectangle`.    Otherwise, it will introduce inconsistencies in future.